buildSrc/src/main/groovy/io.micronaut.build.internal.flyway-module.gradle

```diff
@@ -1,3 +1,9 @@
 plugins {
     id 'io.micronaut.build.internal.module'
 }
+
```

Comment:
Create a base module. see https://github.com/micronaut-projects/micronaut-aws/pull/1445

Reply:
I'm a little confused. This is already follows the example of https://github.com/micronaut-projects/micronaut-aws/pull/1445

Do you mean you want something like this?

file `buildSrc/src/main/groovy/io.micronaut.build.internal.flyway-base.gradle`
```
plugins {
    id 'io.micronaut.build.internal.module'
}

repositories {
    mavenCentral()
    maven { url "https://s01.oss.sonatype.org/content/repositories/snapshots/" }
}

```

file `buildSrc/src/main/groovy/io.micronaut.build.internal.flyway-module.gradle`
```
plugins {
    id 'io.micronaut.build.internal.flyway-base'
}
```

Although I don't think we want `io.micronaut.build.internal.module` in `flyway-base` because we might want to use it to expose the snapshot repos to test sub-projects in our projects that have them, so maybe this?

file `buildSrc/src/main/groovy/io.micronaut.build.internal.flyway-base.gradle`
```
repositories {
    mavenCentral()
    maven { url "https://s01.oss.sonatype.org/content/repositories/snapshots/" }
}

```
file `buildSrc/src/main/groovy/io.micronaut.build.internal.flyway-module.gradle`
```
plugins {
    id 'io.micronaut.build.internal.flyway-base'
    id 'io.micronaut.build.internal.module'
}

This project doesn't have any "x-tests" subprojects, but if it did:

```
file `buildSrc/src/main/groovy/io.micronaut.build.internal.flyway-tests.gradle`
```
plugins {
    id 'io.micronaut.build.internal.flyway-base'
}
```

Reply:
I implemented the latter. If we add test subprojects we can add the `io.micronaut.build.internal.flyway-tests.gradle` script